python/lsst/ts/cbp/__init__.py

```diff
@@ -11,3 +11,4 @@
 from .component import *
 from .mock_server import *
 from .cli import *
+from .config_schema import *
```

Comment:
I gently suggest moving this earlier in the list -- before the CSC -- since it does not depend on anything above and the CSC depends on it.

Not crucial, but makes the import order a bit more obvious.